server/password_test.go

```diff
@@ -355,9 +361,35 @@ func TestSendResetPasswordEmailHandler(t *testing.T) {
 	}
 }
 
+type resetPasswordTestCase struct {
+	userID string
+	query  url.Values
+
+	method   string
+	path     string
+	verified bool
+
+	wantFormValues *url.Values
+	wantCode       int
+	wantPassword   string
+}
+
 func TestResetPasswordHandler(t *testing.T) {
-	makeToken := func(userID, password, clientID string, callback url.URL, expires time.Duration, signer jose.Signer) string {
-		pr := user.NewPasswordReset(user.User{ID: "ID-1"},
+	rpwUserID := "ID-1"
```

Comment:
why all these changes? I don't think it's necessary to prefix all this stuff with "rpw" and "pw" - these variables are local to this file anyhow. In addition, it makes this particular commit larger and harder to read - it's difficult to see where the real changes are.